packages/graphql/package.json

```diff
@@ -35,12 +39,15 @@
     "build": "tsup",
     "check": "gql.tada check",
     "doctor": "gql.tada doctor",
+    "introspect:local": "BASE_URL=http://localhost:3000/graphql ./introspection-curl.sh",
```

Comment:
I would say, let's just inline these in `gql:download:local` and `gql:download:staging` respectively:
```
"gql:download:local": "gql-tada generate schema 'http://localhost:3000/graphql' --output './schema.graphql' && ",
 "gql:download:staging": "gql-tada generate schema 'https://api.staging.lens.dev/graphql' --output './schema.graphql' && BASE_URL=https://api.staging.lens.dev/graphql ./introspection-curl.sh",
```

to get the idea.